main/lsp/LSPMessage.h

```diff
@@ -55,30 +53,15 @@ class LSPMessage final {
     LSPMessage(rapidjson::Document &d);
     LSPMessage(const std::string &json);
 
-    /** A tracer for following LSP message in time traces. May contain multiple tracers if other messages were merged
-     * into this one.*/
-    std::vector<FlowId> startTracers;
-    /** Used to calculate latency of message processing. If this message represents multiple edits, it contains the
-     * oldest timer.*/
-    std::unique_ptr<Timer> timer;
-    /** A more specific timer for the given method. Used to track latency for specific types of requests. */
-    std::unique_ptr<Timer> methodTimer;
-
-    /** If `true`, then this LSPMessage contains a canceled LSP request. */
-    bool isCanceled() const;
```

Comment:
`cancel()` and `isDelayable()` logic has now been moved into the `LSPTask` class.